ShoppingCartTest/ShoppingCartTest.cs

```diff
@@ -0,0 +1,138 @@
+using System;
+using Xunit;
+using ShoppingCartApplication_CleanCodePractices;
```

Comment:
Follow proper project and namespace naming conventions

Reply:
Yes there were multiple projects with the same name so had to name this one like this